src/tests/e2e.test.tsx

```diff
@@ -5,15 +5,26 @@ import {
   Component,
   Fragment,
   Ref,
-  render,
 } from 'preact';
-import {memo, PureComponent, forwardRef} from 'preact/compat';
+import {memo, PureComponent, forwardRef, render} from 'preact/compat';
 import {useState, useEffect, useContext} from 'preact/hooks';
 import {createPortal} from 'preact/compat';
 import {mount, createMount} from '../mount';
 import {ComponentType} from '../types';
 
 describe('@shopify/preact-testing', () => {
+  it('does not time out with large trees', () => {
```

Comment:
This test was fun to write. It fails if we exceed the callstack size, and the entirety of jest crashes if we exceed heap size, so it works well to make sure this keeps working.